### DISCUSSION TITLE
Where does Valent store paired device data?

### DISCUSSION THREAD

**Discussion by xusiai** [andyholmes/valent#580] [Help] [answered]
Where does Valent store paired device data? 

I have root on tmpfs and added $HOME/.cache/valent and $HOME/.config/valent to a permanent state so these folders are not wiped on each boot. However Valent still prompts me to pair a device after each reboot. Is paired device data stored elsewhere?

**Comment by andyholmes** [comment, marked as answer]
Data is indeed stored in `~/.config/valent`, however the paired state is also stored in GSettings. In a Flatpak that will be a keyfile in `~/.config/glib-2.0` (actually under `~/.var`), or for a traditional install in the shared dconf database in `~/.config/dconf`.

Probably I should get around to removing the GSettings key though, I was never really happy with that dual-state.

**Reply by xusiai** [reply]
Thank you very much!
